tests/integration/internal/receivermock/receiver_mock.go

```diff
@@ -0,0 +1,71 @@
+package receivermock
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+	"testing"
+
+	http_helper "github.com/gruntwork-io/terratest/modules/http-helper"
+)
+
+type MetricCounts map[string]int
```

Comment:
Add a comment on this type what does it represent: I assume it's a mapping of metric names to metric counts but who knows :)